### DISCUSSION TITLE
Help regarding connecting through Azure SSO

### DISCUSSION THREAD

**Discussion by OfirAviv8** [iterate-ch/cyberduck#15268] [Q&A]
Hi!
Recently we have been looking for a good S3 file browser solution that works with the SSO we’ve integrated into AWS.
This SSO is using Azure AD as the IdP in order to grant users access to the AWS management console.
Now we have been using and testing Cyberduck just for that, but haven’t really gotten to find and make such option of connection other then the “AWS S3 +STS & Azure Active Directory (Azure AD)” which does use Azure AD as an IdP but we were having trouble figuring out what the “Provide additional login credentials” space is, we did try and insert an ARN for a role which gives access to the bucket but I guess that was not the answer.
	
Thank you for listening and help!

**Comment by dkocher** [comment]
Can you let me know if you followed our [documentation](https://docs.cyberduck.io/protocols/s3/#connecting-with-openid-connect-oidc-identity-provider) and blog post [Connecting with OpenID Connect (OIDC) identity provider](https://docs.cyberduck.io/protocols/s3/#connecting-with-openid-connect-oidc-identity-provider) about setting up the IdP configuration in AWS IAM.

**Reply by dkocher** [reply]
@OfirAviv8 Let us know if you could successfully set this up with the current documentation.
